inst/validation/specs.R

```diff
@@ -468,7 +468,15 @@ roc_module <- list(
         bookmark = "The module shall allow bookmarking"
 )
 
+# common_logic ----
+common_logic <- list(
+  anlfl_col_data_filtering = "
+        The function shall subset a biomarker dataset using the provided category, parameter, visit, and subject selections as well as the specified analysis flag column (anlfl_col).
```

Comment:
I would remove the mention to `anlfl_col` as that belong to the internals of the function and is not relevant for the specs.